src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,1327 @@
+// Distributed under the MIT License.
```

Comment:
This file is really long...any way to factor it?

Reply:
Reduced file length by 35% by moving the intermediates' computation to separate file. Refactoring beyond this will require splitting variable-wise, which does not seem very reasonable to me. Any other ideas are welcome!